src/Streamly/Prelude.hs

```diff
@@ -318,7 +318,7 @@
 -- performance reasons.
 
 module Streamly.Prelude
-{-# DEPRECATED "Please use \"Streamly.Data.Stream.Prelude\" instead." #-}
+{-# DEPRECATED "Please use \"Streamly.Internal.Data.Stream\", \"Streamly.Internal.Data.Stream.Concurrent\", \"Streamly.Internal.Data.Stream.Time\" & \"Streamly.Internal.Data.Stream.Exception\" from streamly-core package instead." #-}
```

Comment:
This is a released module so here we should just suggest released modules `Streamly.Data.Stream.Prelude` and `Streamly.Data.Fold` as we have moved the folds to the Fold module.